packages/server/api/controllers/favorites.controller.js

```diff
@@ -50,9 +52,11 @@ const createFavorites = async (body) => {
 };
 
 // delete
-const deleteFavorites = async (favoritesId) => {
-  if (!favoritesId) {
-    throw new HttpError('please enter ID to delete your favorite.', 400);
+const deleteFavorites = async (token, favoritesId) => {
+  const userUid = token.split(' ')[1];
+  const user = (await knex('users').where({ uid: userUid }))[0];
+  if (!user) {
+    throw new HttpError('User not found', 401);
   }
   try {
     const deletedFav = knex('favorites').where({ id: favoritesId }).del();
```

Comment:
You should also check for the user ID here, otherwise any user can delete any favorite of another user.

Reply:
done ✅